frontends/web/src/locales/de/app.json

```diff
@@ -760,7 +760,7 @@
         "kyc": {
           "link": "Lese die Pocket FAQs",
           "p1": "Für den Handel bis zu 950 EUR (1000 CHF) pro Tag sind keine weiteren Dokumente erforderlich.",
```

Comment:
I believe this is now wrong.

Reply:
We should update the English version and update the daily kyc-light faq.

Reply:
updated in locize

Reply:
pulled